test-utils/src/main/kotlin/com/google/devtools/ksp/processor/SealedClassProcessor.kt

```diff
@@ -31,7 +31,7 @@ class SealedClassProcessor : AbstractTestProcessor() {
         resolver.getNewFiles().forEach { f ->
             f.declarations.forEach {
                 if (it is KSClassDeclaration) {
-                    val subs = it.getSealedSubclasses().map { it.simpleName.asString() }.toList()
+                    val subs = it.getSealedSubclasses().map { it.simpleName.asString() }.toList().sorted()
```

Comment:
Just curious; why wasn't it sorted before, and why is it sorted now? :)

Reply:
Because of the dependency to compiler version, behavior of KSP API's returning order can actually change if there is an upstream change, and therefore we will just use sort() to get rid of the checking on the ordering if it doesn't matter.